sno/working_copy/postgis.py

```diff
@@ -609,3 +652,96 @@ def _update_state_table_tree_impl(self, dbcur, tree_id):
             (tree_id,),
         )
         return changes_rowcount(dbcur)
+
+    def _is_meta_update_supported(self, dataset_version, meta_diff):
+        """
+        Returns True if the given meta-diff is supported *without* dropping and rewriting the table.
+        (Any meta change is supported if we drop and rewrite the table, but of course it is less efficient).
+        meta_diff - DeltaDiff object containing the meta changes.
+        """
+        if not meta_diff:
+            return True
+
+        if "schema.json" not in meta_diff:
+            return True
+
+        schema_delta = meta_diff["schema.json"]
+        if not schema_delta.old_value or not schema_delta.new_value:
+            return False
+
+        old_schema = Schema.from_column_dicts(schema_delta.old_value)
+        new_schema = Schema.from_column_dicts(schema_delta.new_value)
+        dt = old_schema.diff_type_counts(new_schema)
+
+        # We support deletes, name_updates, and type_updates -
+        # but we don't support any other type of schema update except by rewriting the entire table.
+        dt.pop("deletes")
+        dt.pop("name_updates")
+        dt.pop("type_updates")
+        return sum(dt.values()) == 0
+
+    def _apply_meta_title(self, dataset, src_value, dest_value, dbcur):
+        pass  # This is a no-op for postgis
+
+    def _apply_meta_description(self, dataset, src_value, dest_value, dbcur):
+        pass  # This is a no-op for postgis
+
+    def _apply_meta_metadata_dataset_json(self, dataset, src_value, dest_value, dbcur):
+        pass  # This is a no-op for postgis
```

Comment:
these will be the norm - most other databases won't support these either. Move these to base.py?

Reply:
Still think we should use `COMMENT ON TABLE {mydataset} IS 'the dataset title';` 😄

Reply:
Done